swagger/redhatopenshift/resource-manager/Microsoft.RedHatOpenShift/stable/2020-04-30/redhatopenshift.json

```diff
@@ -735,7 +735,11 @@
         "Standard_F4s_v2",
         "Standard_F8s_v2"
       ],
-      "type": "string"
+      "type": "string",
+      "x-ms-enum": {
+        "name": "VMSize",
+        "modelAsString": true
+      }
     },
     "Visibility": {
       "description": "Visibility represents visibility.",
```

Comment:
don't you want the x-ms-enum here too? or why only vmsize?

Reply:
I do not understand the question, the enum is here for defined vmsize. The modelasstring makes it not requiring the value. the original enum stays the same as it is present for the receivers not knowing about extensions.

Did i get it right?